ambry-server/src/test/java/com.github.ambry.server/MockCluster.java

```diff
@@ -47,17 +47,17 @@
   private NotificationSystem notificationSystem;
   private boolean serverInitialized = false;
 
-  public MockCluster(NotificationSystem notificationSystem, Time time)
+  public MockCluster(NotificationSystem notificationSystem, boolean enableHardDeletes, Time time)
       throws IOException, InstantiationException, URISyntaxException, GeneralSecurityException {
-    this(notificationSystem, false, "", new Properties(), true, time);
+    this(notificationSystem, false, "", new Properties(), enableHardDeletes, time);
   }
 
   public MockCluster(NotificationSystem notificationSystem, boolean enableSSL, String datacenters, Properties sslProps,
       boolean enableHardDeletes, Time time)
       throws IOException, InstantiationException, URISyntaxException, GeneralSecurityException {
     // sslEnabledDatacenters represents comma separated list of datacenters to which ssl should be enabled
     this.notificationSystem = notificationSystem;
-    clusterMap = new MockClusterMap(enableSSL);
+    clusterMap = new MockClusterMap(enableSSL, 9, 3, 3);
```

Comment:
Should this variable name be `mockClusterMap` instead of `clusterMap` for consistency with the hard delete test.